pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1733,10 +1792,10 @@
           (try-next-sponsor sponsor.peer-state)
         ::
         ?~  route=route.peer-state
-          %-  (trace rot.veb |.("no route to:  {<ship>}"))
+          %-  (trace rot.veb ship |.("no route to:  {<ship>}"))
```

Comment:
I think this should be gated on the original `ship`.  If I want to trace the routes tried to send to `~wicdev-wisryt`, I don't want that not printing that it tried `~marzod` just because I didn't list it.  You'll need to store the original ship at the top of +send-blob for this purpose.

Same below